lib/msal-node/src/cache/TokenCache.ts

```diff
@@ -124,7 +124,7 @@ export class TokenCache implements ISerializableTokenCache, ITokenCache {
         let cacheContext;
         try {
             if (this.persistence) {
-                cacheContext = new TokenCacheContext(this, false);
+                cacheContext = new TokenCacheContext(this, true);
```

Comment:
This is necessary because `getAllAccounts` is now a read/write operation. Setting the cache context here works differently than expected.

I expected the cacheContext.hasChanged property to be updated automatically upon CRUD on the cache, but it doesn't. Setting the cacheContext like this before and after the operation seems to function as a declaration of whether the cache should be updated after the operation (i.e. passing in false means the cache shouldn't update, passing in true means it should).